src/types/Errors.ts

```diff
@@ -0,0 +1,5 @@
+export enum Errors {
+  DEFAULT = '',
```

Comment:
```suggestion
export enum Error {
  NONE = '',
```
Use singular form for enum names
Maybe it's better to rename it to `NONE`, WDYT? Because "default error" sounds a bit misleading